src/vsphere_cpi/spec/unit/cloud/vsphere/drs_rules/drs_rule_spec.rb

```diff
@@ -151,4 +151,48 @@ def self.it_adds_custom_attribute_to_vm
     end
   end
 
+  describe '#add_vm_host_affinity_rule' do
+    let(:vm) { instance_double('VimSdk::Vim::VirtualMachine', name: 'fake-vm') }
+    let(:fake_vm_1) { instance_double('VimSdk::Vim::VirtualMachine', name: 'fake-vm-1') }
```

Comment:
vm, fake_vm_1 can all be removed once the add_vm_host_affinity_rule definition is updated